source/user-manual/reference/ossec-conf/global.rst

```diff
@@ -90,7 +90,7 @@ This specifies the email recipient for alerts.
 | **Allowed values** | Any valid email address |
 +--------------------+-------------------------+
 
-This section will only allow for one email address, but the section can be repeated for each email address you would wish include.
+This section will only allow for one email address, but the section can be repeated for each email address you would wish to include.
```

Comment:
```suggestion
Please note that this section only allows for one email address. In case you want to add more than one, repeat the section as many times as required.
```